### ISSUE TITLE
Redis not mentioned or installed ~> errors

### ISSUE THREAD

**Issue by onyoo** [learn-co-curriculum/rails-sidekiq-readme#2]
While trying to run `bundle exec sidekiq` as asked to in the lesson, I was met with this error:

```
[21:40:36] (master) rails-sidekiq-readme-v-000
// ♥ bundle exec sidekiq


         m,
         `$b
    .ss,  $$:         .,d$
    `$$P,d$P'    .,md$P"'
     ,$$$$$bmmd$$$P^'
   .d$$$$$$$$$$P'
   $$^' `"^$$$'       ____  _     _      _    _
   $:     ,$$:       / ___|(_) __| | ___| | _(_) __ _
   `b     :$$        \___ \| |/ _` |/ _ \ |/ / |/ _` |
          $$:         ___) | | (_| |  __/   <| | (_| |
          $$         |____/|_|\__,_|\___|_|\_\_|\__, |
        .d$$                                       |_|

2016-02-24T20:40:46.131Z 17277 TID-ovqk0at50 INFO: Running in ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin15]
2016-02-24T20:40:46.131Z 17277 TID-ovqk0at50 INFO: See LICENSE and the LGPL-3.0 for licensing details.
2016-02-24T20:40:46.131Z 17277 TID-ovqk0at50 INFO: Upgrade to Sidekiq Pro for more features and support: http://sidekiq.org
2016-02-24T20:40:46.131Z 17277 TID-ovqk0at50 INFO: Booting Sidekiq 4.1.0 with redis options {:url=>nil}
Error connecting to Redis on 127.0.0.1:6379 (Errno::ECONNREFUSED)
```

SO was giving no clear answers, but luckily I had watched the awesome Railscast on sidekiq. The problem for me at least was that I didn't have redis installed. This is very important since sidekiq relies on it to manage it's job que, but doesn't download it with `bundle install` :/ Source (around 1:20):http://media.railscasts.com/assets/episodes/videos/366-sidekiq.mp4

Fix (feel free to change or delete the description if it is inaccurate):

We need to install one more gem before sidekiq is ready to tackle our jobs for us in the background. We can get this gem from Homebrew:

```
brew install redis
```

Redis manages tasks on threads. Don't worry about threads too much, just know that they are like parallel roads were different information can run through Ruby a the same time. If one road has heavy traffic, it doesn't affect the other roads.

**Comment by MitulMistry** [comment]
I had the same problem. On Nitrous I had to do: `sudo apt-get install redis-server`

**Comment by pletcher** [comment]
Thanks for commenting, all! Fixed in 163decf.
